client/js/components/iframe-container.js

```diff
@@ -0,0 +1,39 @@
+export default (el) => {
+  const iframeTrigger = el.querySelector('.js-iframe-trigger');
+  const originalIframe = el.querySelector('.js-iframe');
+  const launch = el.querySelector('.js-iframe-launch');
+  const originalLaunchText = launch.innerHTML;
+  const close = el.querySelector('.js-iframe-close');
+
+  iframeTrigger.addEventListener('click', loadIframe);
+  close.addEventListener('click', unloadIframe);
+
+  function loadIframe(event) {
```

Comment:
Do these functions need to be created every time?
If so, can they not just be inline logic?

Reply:
Looking at it, they can't as they rely on the `const`s before.